app/controllers/propertyLinking/UploadController.scala

```diff
@@ -43,7 +43,7 @@ class UploadController @Inject()(
                                   val errorHandler: CustomErrorHandler,
                                   authenticatedAction: AuthenticatedAction,
                                   withLinkingSession: WithLinkingSession,
-                                  businessRatesAttachmentsServices: BusinessRatesAttachmentService
+                                  businessRatesAttachmentsService: BusinessRatesAttachmentService
```

Comment:
i think now the name of the parameter is good `businessRatesAttachmentsService` but the name of the service should be plural "Attachments" not "Attachment" (BusinessRatesAttachment**s**Service)

Reply:
cool, will do that. Will do the same for the connector as well